### PR TITLE
Restore Paladin REs to the Combined Shining Oath Feat

### PR DESCRIPTION
**Pull request by avagdumortesin**
Closes https://github.com/foundryvtt/pf2e/issues/11597.

Also re-input the feat's text from CRBv4 printing and put the benefits for the various causes in the order of that description.

### CODE REVIEW

**Comment by stwlam** [comment]
Check errata

**Comment by avagdumortesin** [comment]
Eye r dum and missed that; still rearranged the order of the various bonuses.

File: packs/feats/shining-oath.json [hunk 1]
```diff
@@ -31,7 +28,45 @@
...
+            {
+                "key": "AdjustModifier",
+                "mode": "add",
+                "predicate": [
+                    {
+                        "or": [
+                            "item:proficiency:rank:3",
+                            "item:proficiency:rank:4"
+                        ]
+                    }
```
Comment (TikaelSol) [inline packs/feats/shining-oath.json:63]:
```suggestion
                    {
                        "gte": [
                            "item:proficiency:rank",
                            3
                        ]
                    }
```
Protects against some horrible future where Paizo drops level 20+ play on us all.

File: packs/feats/shining-oath.json [hunk 2]
```diff
@@ -31,7 +28,45 @@
...
+            {
+                "domain": "damage",
+                "key": "RollOption",
+                "label": "Retributive Strike against an undead",
+                "option": "shining-oath",
+                "predicate": [
+                    "feature:paladin"
+                ],
+                "toggleable": true
+            },
```
Comment (TikaelSol) [inline packs/feats/shining-oath.json:41]:
```suggestion
```
Ugh, this should just be on all retributive strikes, and that should have a toggle on it to kick on automation like this or divine smite.

File: packs/feats/shining-oath.json [hunk 3]
```diff
@@ -31,7 +28,45 @@
...
+            {
+                "key": "FlatModifier",
+                "label": "Shining Oath",
+                "predicate": [
+                    "shining-oath",
+                    "target:mode:undead"
+                ],
```
Comment (TikaelSol) [inline packs/feats/shining-oath.json:48]:
```suggestion
                "predicate": [
                    "retributive-strike",
                    "feature:paladin",
                    "target:mode:undead"
                ],
```
I'll push up a PR to add the retributing strike toggle for paladins

**Pull request merged**